apps/web/Containerfile

```diff
@@ -4,19 +4,22 @@ LABEL maintainer="team@penumbralabs.xyz"
 
 # provide pnpm globally for dep installing and building
 FROM alpine AS base
+ENV NEXT_TELEMETRY_DISABLED=1
```

Comment:
Porting this no-telem env var across all build stages...

Reply:
lol, great catch. Very obnoxious to keep track of.